cpp/models/abm/trip_list.h

```diff
@@ -38,7 +34,7 @@ namespace abm
  * @brief A trip describes a migration from one Location to another Location.
  */
 struct Trip {
-    uint32_t person_id; /**< Person that makes the trip and corresponds to the index into the structure m_persons from
+    PersonId person_id; /**< Person that makes the trip and corresponds to the index into the structure m_persons from
```

Comment:
```suggestion
    PersonId person_id; /**< PersonID of the Person that makes the trip and corresponds to the index into the structure m_persons from
```

Reply:
I don't think repeating the type is helpful in these comments. They should explain/define what the variable does, using its type for that seems tautological. Also, its type is (usually) explicitly written right next to its name and definition anyways.

Although, if you think the comment lacks some important detail, we can update it.

Reply:
Ok, I guess the part "corresponds to the index into the structure m_persons from" could be also a hint to the variable type already.